t/lib/CRMS/RightsPredictor.t

```diff
@@ -51,19 +59,27 @@ subtest 'RightsPredictor::last_source_copyright' => sub {
     is($res->{last_source_copyright_year}, 2042);
   };
 
-  subtest 'Canada post-1972' => sub {
+  subtest 'Canada post-1972 author death date' => sub {
     my $f008 = '850423s1940    cn a          000 0 eng d';
     my $record = FakeMetadata::fake_record_with_008_and_leader($f008);
     my $rp = CRMS::RightsPredictor->new(record => $record);
     my $res = $rp->last_source_copyright(2000);
     is($res->{last_source_copyright_year}, 2070);
   };
 
-  subtest 'Canada crown' => sub {
+  subtest 'Canada post-1972 corporate work' => sub {
     my $f008 = '850423s1940    cn a          000 0 eng d';
     my $record = FakeMetadata::fake_record_with_008_and_leader($f008);
     my $rp = CRMS::RightsPredictor->new(record => $record);
     my $res = $rp->last_source_copyright(2000, 1);
+    is($res->{last_source_copyright_year}, 2075);
+  };
+
+  subtest 'Canada crown' => sub {
```

Comment:
I find these tests hard to read. 
A one sentence comment per test might be helpful for the poor reader.
```
# because $field is $value we expect result to be 2075 (x+y)
```
... or something like that. I've often found that helpful when writing tests too.

Reply:
Added parameters to the `is` tests for `last_source_copyright` to explain what's going on. Added more tests. I'll deal with documenting the `rights` tests on the upcoming DEV ticket.